src/components/Cards/ImageCardCompact.tsx

```diff
@@ -18,22 +18,23 @@ import {rem} from 'rn-units';
 
 type Props = {
   title: string;
-  description: string;
-  imageSource: ImageSourcePropType;
+  description: string | ReactNode;
+  image: ImageSourcePropType;
```

Comment:
```suggestion
  image: ImageSourcePropType | undefined;
```

Have you tested how a badge is displayed if there is no local image for it? E.g. when a user didn't update the app but BE returns some new badge?